docs/resources/cse_microservice_instance.md

```diff
@@ -0,0 +1,178 @@
+  Changing this will create a new microservice instance.
+
+* `data_center` - (Optional, List, ForceNew) Specifies the data center configuration.
+  The [object](#microservice_instance_data_center) structure is documented below.
+  Changing this will create a new microservice instance.
+
+* `admin_user` - (Optional, String, ForceNew) Specifies the account name. The initial account name is **root**.
+  Required if the `auth_type` of engine is **RBAC**. Changing this will create a new microservice instance.
+
+-> Please make sure that the public IP has been bound to the Engine before use.
+
+* `admin_pass` - (Optional, String, ForceNew) Specifies the account password.
+  Required if the `auth_type` of engine is **RBAC**. Changing this will create a new microservice instance.
+  The password format must meet the following conditions:
+  + Must be `8` to `32` characters long.
+  + A password must contain at least one digit, one uppercase letter, one lowercase letter, and one special character
+    (-~!@#%^*_=+?$&()|<>{}[]).
+  + Cannot be the account name or account name spelled backwards.
+  + The password can only start with a letter.
+
+<a name="microservice_instance_health_check"></a>
+The `health_check` block supports:
+
+* `mode` - (Required, String, ForceNew) Specifies the heartbeat mode. The valid values are **push** and **pull**.
+  Changing this will create a new microservice instance.
+
+* `interval` - (Required, Int, ForceNew) Specifies the heartbeat interval. The unit is **s** (second).
+  Changing this will create a new microservice instance.
+
+* `max_retries` - (Required, Int, ForceNew) Specifies the maximum retries.
+  Changing this will create a new microservice instance.
+
+* `port` - (Optional, Int, ForceNew) Specifies the port number.
+  Changing this will create a new microservice instance.
+
+<a name="microservice_instance_data_center"></a>
+The `data_center` block supports:
+
+* `name` - (Required, String, ForceNew) Specifies the data center name.
+  Changing this will create a new microservice instance.
+
+* `region` - (Required, String, ForceNew) Specifies the custom region name of the data center.
+  Changing this will create a new microservice instance.
+
+* `availability_zone` - (Required, String, ForceNew) Specifies the custom availability zone name of the data center.
+  Changing this will create a new microservice instance.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The microservice instance ID.
+
+* `status` - The microservice instance status. The values supports **UP**, **DOWN**, **STARTING** and **OUTOFSERVICE**.
+
+## Import
+
+Microservices can be imported using related `connect_address`, `microservice_id` and their `id`, separated by a
+slash (/), e.g.
+
+```
+$ terraform import huaweicloud_cse_microservice_instance.test https://124.70.26.32:30100/f14960ba495e03f59f85aacaaafbdef3fbff3f0d
```

Comment:
the instance id is missing in the example

Reply:
done.